tsconfig.json

```diff
@@ -1,11 +1,11 @@
 {
 	"compilerOptions": {
-		"module": "commonjs",
+		"module": "NodeNext",
 		"target": "es2021",
 		"lib": ["es2021"],
 		"sourceMap": true,
 		"declaration": true,
-		"moduleResolution": "node16",
+		"moduleResolution": "NodeNext",
 		"alwaysStrict": true,
```

Comment:
Also fixing this inconsistency. commonjs module is actually incompatible with node16 moduleResolution. So at the minimum they both shold be node16. With that said, NodeNext is recommended over node16 although currently NodeNext still uses node16. Node 20 is our goal - so its best to just put in NodeNext now since it'll auto resolve as things move in that direction I believe.